tests/virt_autotest/host_upgrade_step2_run.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+use base "host_upgrade_base";
+#use virt_utils qw(set_serialdev);
+use testapi;
+use strict;
+
+sub get_script_run() {
+    my $self = shift;
+
+    my $pre_test_cmd = $self->get_test_name_prefix;
+    $pre_test_cmd .= "-run 02";
+
+    return "$pre_test_cmd";
+}
+
+sub run() {
+    my $self = shift;
+
+    # Set the correct serial dev for ipmi xen and non-xen host according to the installed product release
+    # &virt_utils::set_serialdev();
+
+    # Got script run according to different kind of system
+    my $pre_test_cmd = $self->get_script_run();
+
+    # Execute script run
+    my $ret = $self->execute_script_run($pre_test_cmd, 36000);
```

Comment:
Thanks for the explanation.
The test time in our test depends on how many guests you want to test in one round. It certainly can be split into smaller tests so as to finish in shorter time. And we do plan to use this way for the real testing when deployed in official openqa server. And we use physical machine for testing, so it is not able to use assets because recovering the back up disk on physical machine takes longer time than reinstall the host(10 to 20 minutes). And the START_AFTER works in some of our tests which does not pollute environment, while some can not because the test environment changes after one round test(host upgrade).